go/vt/tabletmanager/gorpctmserver/gorpc_server.go

```diff
@@ -35,6 +36,7 @@ type TabletManager struct {
 
 // Ping wraps RPCAgent.
 func (tm *TabletManager) Ping(ctx context.Context, args, reply *string) error {
+	ctx = callinfo.RPCWrapCallInfo(ctx)
```

Comment:
It is needed if the server uses the CallInfo extra data (otherwise, no need to add it, it's just extra work for nothing).

The only two that do for now are:
- sqlquery
- tmserver
I think I covered both in this CL.